test/utils/AddressHelperSpec.scala

```diff
@@ -17,24 +17,52 @@
 package utils
 
 import base.SpecBase
-import models.{Address, AddressLookup, Country}
+import models.{Address, AddressLookup, AddressResponse, Country}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
 
 class AddressHelperSpec extends SpecBase {
   val sut = AddressHelper
 
   "formatAddress" - {
-    "format the address correctly" in {
+    "format Address correctly" in {
       val address = Address("line1", Some("line2"), "line3", Some("line4"), Some("postcode"), Country.GB)
       val result  = sut.formatAddress(address)
       result mustBe "line1, line2, line3, line4, postcode, United Kingdom"
     }
+    "format Address correctly as html" in {
+      val address = Address("line1", Some("line2"), "line3", Some("line4"), Some("postcode"), Country.GB)
+      val expectedResult = HtmlContent(
+        """<p class='govuk-!-margin-top-0 govuk-!-margin-bottom-0'>line1</p>"""
+          + """<p class='govuk-!-margin-top-0 govuk-!-margin-bottom-0'>line2</p>"""
+          + """<p class='govuk-!-margin-top-0 govuk-!-margin-bottom-0'>line3</p>"""
+          + """<p class='govuk-!-margin-top-0 govuk-!-margin-bottom-0'>line4</p>"""
+          + """<p class='govuk-!-margin-top-0 govuk-!-margin-bottom-0'>postcode</p>"""
+          + """<p class='govuk-!-margin-top-0 govuk-!-margin-bottom-0'>United Kingdom</p>"""
+      )
```

Comment:
This can be changed to something like this:
```
      val expectedResult = HtmlContent(
        """
          |<p class='govuk-!-margin-top-0 govuk-!-margin-bottom-0'>line1</p>
          |<p class='govuk-!-margin-top-0 govuk-!-margin-bottom-0'>line2</p>
          |<p class='govuk-!-margin-top-0 govuk-!-margin-bottom-0'>line4</p>
          |<p class='govuk-!-margin-top-0 govuk-!-margin-bottom-0'>postcode</p>
          |<p class='govuk-!-margin-top-0 govuk-!-margin-bottom-0'>United Kingdom</p>
          |""".stripMargin
      )
      ```